Makefile

```diff
@@ -105,18 +105,21 @@ build: ## By default we should build in release mode
 
 .PHONY: build-no-std
 build-no-std: ## Build without the standard library
-	${BUILD_KERNEL_ERRORS} cargo build --no-default-features --target wasm32-unknown-unknown --workspace --lib
+	${BUILD_KERNEL_ERRORS} cargo build --no-default-features --target wasm32-unknown-unknown --workspace --lib --exclude miden-proving-service --features tx-prover
 
 
 .PHONY: build-no-std-testing
 build-no-std-testing: ## Build without the standard library. Includes the `testing` feature
-	cargo build --no-default-features --target wasm32-unknown-unknown --workspace --exclude miden-bench-tx --exclude miden-tx-prover --features testing
+	cargo build --no-default-features --target wasm32-unknown-unknown --workspace --exclude miden-bench-tx --exclude miden-proving-service --features testing,tx-prover
 
 
 .PHONY: build-async
 build-async: ## Build with the `async` feature enabled (only libraries)
 	${BUILD_KERNEL_ERRORS} cargo build --lib --release --features async
 
+.PHONY: build-remote-provers
+build-remote-provers: ## Build the remote provers
+	${BUILD_KERNEL_ERRORS} cargo build --release --lib --features tx-prover -p miden-remote-provers
```

Comment:
nit: Since this is already just a lib, we could remove the `--lib` here.

But is this target neeed? This should be covered by the `build` targets, right?

Reply:
Agree that it is covered, I'm removing it.